packages/frontend/components/composed/article/HowToCite/HowToCite.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState, useEffect } from "react";
+import { graphql } from "react-relay";
+import { useMaybeFragment } from "@wdp/lib/api/hooks";
+import { useTranslation } from "react-i18next";
+import ReactMarkdown from "react-markdown";
+import { remark } from "remark";
+import strip from "strip-markdown";
+import { Button } from "components/atomic";
+import { HowToCiteFragment$key } from "@/relay/HowToCiteFragment.graphql";
+import * as Styled from "./HowToCite.styles";
+
+export default function HowToCite({ data }: Props) {
+  const result = useMaybeFragment(fragment, data);
+  const citation = result?.citation?.content;
+  const { t } = useTranslation();
+  const [plainText, setPlainText] = useState("");
+
+  useEffect(() => {
+    remark()
+      .use(strip)
+      .process(citation || "")
+      .then((value) => {
+        setPlainText(String(value));
+      });
+  }, [citation]);
```

Comment:
@jen-castiron Do you have a better way to strip the markdown for copying? I don't think this preserves bold/italics, so it's not ideal.

Reply:
How about using a ref on the ReactMarkdown element, and then when the button is clicked, copy the ref element's content? That should preserve styling and links.

Reply:
I tried that first. `ReactMarkdown` doesn't accept a ref. 😞

Reply:
You can always wrap the `ReactMarkdown` component in a div and use that as a ref! I'm doing something similar in ArticleText, in order to get all of the headers after the text has rendered.